packages/obytelib/obytelib.1.6/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+authors: ["Benoît Vaugon"]
+homepage: "https://github.com/bvaugon/obytelib"
+bug-reports: "https://github.com/bvaugon/obytelib/issues"
+dev-repo: "git+https://github.com/bvaugon/obytelib.git"
+maintainer: "benoit.vaugon@gmail.com"
+license: "CeCILL-2.0"
+build: [
+  ["./configure" "-prefix" prefix]
+  [make "all"]
+]
+remove: [
+  ["./configure" "-prefix" prefix]
+  [make "uninstall"]
+]
```

Comment:
```suggestion
```
Are there other file to remove than the one that were installed? If not the remove field is unnecessary